### DISCUSSION TITLE
Welcome to the Markdoc community!

### DISCUSSION THREAD

**Discussion by mfix-stripe** [markdoc/markdoc#1] [Announcements]
## 👋 Welcome!
Hey you. We’re using Discussions as a place to connect with folks that are interested in Markdoc. We hope that you:
* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we build together.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by orliesaurus** [comment]
Thank you for releasing this - can't wait to give this a try!

**Comment by raymondworkshop** [comment]
Thanks, and how to join the contributions?

**Reply by mfix-stripe** [reply]
Hey @muyun, welcome! If you are interested in making contributions, feel free to [open a Pull Request](https://makeapullrequest.com/) on any of our open-source repos.
